src/clr-addons/components.clr-addons.scss

```diff
@@ -5,37 +5,60 @@
 @import './view-edit-section/view-edit-section';
 @import './content-panel/content-panel';
 @import './breadcrumb/breadcrumb';
+@import './flow-bar/flow-bar';
 
-$clr-card-active-border-color: rgba(0, 148, 210, 1) !default;
-$clr-card-active-border-width: 0.25rem !default;
+$clr-card-active-box-shadow-color: rgba(0, 148, 210, 1) !default;
 
 .content-header {
-  border-bottom: $clr-global-borderwidth solid $clr-light-midtone-gray;
-  padding: 0.5rem 1rem 0.25rem 1rem;
+  padding: 0.5rem 1rem 0 1rem;
   display: flex;
-  flex-shrink: 0;
-}
+  flex: 0 0 auto;
 
-.content-header > h2 {
-  margin-top: 0;
-}
+  h2 {
```

Comment:
we should not overwrite h2 globally, as also h2's in content will get their margin-top set to 0

Reply:
this is nested inside .content-header